app/src/main/java/org/simple/clinic/summary/PatientSummaryEffect.kt

```diff
@@ -64,6 +64,8 @@ data class CalculateCVDRisk(val patient: Patient) : PatientSummaryEffect()
 
 data class LoadStatinInfo(val patientUuid: UUID) : PatientSummaryEffect()
 
+data class UpdateSmokingStatus(val patientId: UUID, val isSmoker: org.simple.clinic.medicalhistory.Answer) : PatientSummaryEffect()
```

Comment:
Can we use an import alias for the `Answer`, something like `MedicalHistoryAnswer`, wdyt?

Reply:
We also have `import org.simple.clinic.patient.Answer` in the same class that's why it's being used like here.

Reply:
Yeah. That's why I was suggesting import aliases

https://kotlinlang.org/docs/packages.html#imports

We can use 'as' keyword